vector-im/element-android
# Space Switching Back Navigation

## Type of change


## Content

Adds a space backstack that gets used when navigating back with the android back button.

## Motivation and context

Closes https://github.com/vector-im/element-android/issues/6751

## Screenshots / GIFs

https://user-images.githubusercontent.com/20701752/185617808-42abb9b0-b0ac-4d5f-a8ce-e9f052a2ccaf.mp4

## Tests

- Navigate through spaces and go back using the device back button. See that you can go back and forward seamlessly
- See that going back when in All Chats minimises the app

## Tested devices

- OS version(s): Android 12

## Checklist
